### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by SidSethi**
- Tried to make this dead simple and only include whats relevant for a node operator
- also removed redundant info from [Content Node Setup Guide](https://docs.earthfast.com/node-operators/content-node-setup)

### CODE REVIEW

File: content-node/docker-compose/README.md [hunk 1]
```diff
@@ -2,71 +2,58 @@
 
 This repository contains the necessary files to deploy a Content Node using Docker Compose.
 
-## Prerequisites
+This README assumes you have followed the [Content Node Setup Guide](https://docs.earthfast.com/node-operators/content-node-setup) and satisfied all prerequisites & registered your Content Node.
 
-- A Linux-based system (preferably Ubuntu)
```
[the remark this answers is not in the document]
Comment (dmanjunath) [reply content-node/docker-compose/README.md:7]: i think this is covered in the docs site. having one source of truth is good. but i agree the docs should explicitly say ubuntu

File: content-node/docker-compose/README.md [hunk 2]
```diff
@@ -2,71 +2,58 @@
...
-./setup.sh --staging
-```
-
-##### SSL Configuration
 
-If `SETUP_SSL` is set to 'true', the setup will automatically configure SSL using Let's Encrypt. Make sure the url from `SERVER_NAME` correctly pointed to your server's IP address before enabling this option.
+The `setup.sh` script will prompt you for environment variables:
+* `SERVER_NAME` = your node's FQDN
+* `NODE_ID` = the NodeID outputted in Step 5 above
+* `SETUP_SSL` = if you have configured this externally, set this to false. Setting it to true will locally configure SSL with Let's Encrypt. Make sure the url from SERVER_NAME is correctly pointed to your server's IP address before enabling this option.
```
Comment (dmanjunath) [inline content-node/docker-compose/README.md:19]: "if you have configured HTTPS/SSL externally, set this to false."

File: content-node/docker-compose/README.md [hunk 3]
```diff
@@ -2,71 +2,58 @@
...
+> ./setup.sh
 ```
-./setup.sh --staging
-```
-
-##### SSL Configuration
 
-If `SETUP_SSL` is set to 'true', the setup will automatically configure SSL using Let's Encrypt. Make sure the url from `SERVER_NAME` correctly pointed to your server's IP address before enabling this option.
+The `setup.sh` script will prompt you for environment variables:
+* `SERVER_NAME` = your node's FQDN
```
Comment (dmanjunath) [inline content-node/docker-compose/README.md:17]: lets give an example

File: content-node/docker-compose/README.md (hunk 2 above)
Comment (dmanjunath) [reply content-node/docker-compose/README.md:19]: if a user selects SETUP_SSL=true, we should check in setup.py if the FQDN points to this server. not a blocker to merge this PR but probably good developer experience

File: content-node/docker-compose/README.md [hunk 4]
```diff
@@ -2,71 +2,58 @@
...
+* `NODE_ID` = the NodeID outputted in Step 5 above
```
Comment (dmanjunath) [inline content-node/docker-compose/README.md:18]: lets add a link to this? https://docs.earthfast.com/node-operators/content-node-setup#register-your-node-s-onchain

**Review by dmanjunath — APPROVED** [review]
small suggestions about env var clarity

File: content-node/docker-compose/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SidSethi) [reply content-node/docker-compose/README.md:7]: 1) why ubuntu specifically vs debian based?
2) yeh wanted to have all info in one place only, can still see an arg to put this all in the docsite, but felt the code specific instructions made more sense here?

File: content-node/docker-compose/README.md (hunk 2 above)
Comment (SidSethi) [reply content-node/docker-compose/README.md:19]: updated readme, added [linear card](https://linear.app/earthfast/issue/ENG-234/if-setup-ssl=true-check-in-setuppy-if-fqdn-points-to-vm-ip)

File: content-node/docker-compose/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cheran-senthil) [reply content-node/docker-compose/README.md:7]: Saying Debian should be fine, but I haven't tested Debian yet, in theory, I see no issues.

File: content-node/docker-compose/README.md (hunk 3 above)
Comment (SidSethi) [reply content-node/docker-compose/README.md:17]: fixed

File: content-node/docker-compose/README.md (hunk 4 above)
Comment (SidSethi) [reply content-node/docker-compose/README.md:18]: whoops, fixed

**Review by cheran-senthil — APPROVED** [review]
Made changes to support other operating systems

File: content-node/docker-compose/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SidSethi) [reply content-node/docker-compose/README.md:7]: sounds like this is supported now

**Pull request merged**